index.html

```diff
@@ -222,5 +222,33 @@ <h1 class="greet">
           </ul>
       </div>
     </main>
+    <footer class="contact-container">
+      <div class="contact-bg">
+      <div class="contacts">Contact me</div>
+      <div class="contacts-txt">
+        If you have an application you
+        are intrested in developing, a feature
+        that you need to build or a project that needs
+        coding . I'd love to help with it 
+      </div>
+      <form method="post" action="https://formspree.io/f/{form_id}">
+        <ul class="contact-form">
+          <li class="name-form">
+            <label for="name"></label>
+            <input type="text" id="name" name="user_name" placeholder="Name" required/>
```

Comment:
- [ ] Based on the required features in the [Project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), the name filed should have a `maxlength="30"` attribute. please do make the change 👍 

![image](https://user-images.githubusercontent.com/50990925/213105913-4c7a2407-293f-4831-a5ae-a9ea52eef473.png)